lib/OpenQA/Worker/Commands.pm

```diff
@@ -88,15 +88,16 @@ sub websocket_commands {
     elsif ($msg eq 'livelog_start') {
         # change update_status timer if $job running
         if (backend_running) {
-            $OpenQA::Worker::Jobs::do_livelog = 1;
-            change_timer('update_status', STATUS_UPDATES_FAST);
+            print "starting livelog\n" if $verbose;
+            $OpenQA::Worker::Jobs::do_livelog += 1;
```

Comment:
assuming that the status update does noops most of the time, perhaps we can lower the SLOW updates